sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -25,11 +29,11 @@ FlashPredict::FlashPredict(fhicl::ParameterSet const& p)
   , fFlashEnd(p.get<double>("FlashEnd"))  // in us w.r.t flash time
   , fTimeBins(timeBins())
   , fSelectNeutrino(p.get<bool>("SelectNeutrino", true)) // only attempt to match potential neutrino slices
-  , fOnlyCollectionWires(p.get<bool>("OnlyCollectionWires", true))
+  , fPlaneList(p.get<std::vector<int>>("PlaneList")) // Use 0, 1, 2 for SBND and 0, 1, 3 for ICARUS
```

Comment:
Better to put this line after the `fSBND` and `fICARUS` and then assign `kSBNDPlanes` or `kICARUSPlanes` as defaults.